server/src/model/user.entity.ts

```diff
@@ -84,11 +85,15 @@ export class User {
   group!: IdentifiedReference<Group>;
 
   /** Event trackers for each event the player participated in */
-  @OneToMany(() => EventTracker, ev => ev.user)
+  @OneToMany(() => EventTracker, ev => ev.user, {
+    cascade: [Cascade.SCHEDULE_ORPHAN_REMOVAL, Cascade.REMOVE],
```

Comment:
I really like how the property cascade was taken advantage of. I didn't realize that there was this function that allows you to remove elements from the backend so easily. I like how instead of trying to remember all references to the deleted element, you can implement cascade and essentially have the database do the work for you. Very nice!